### PR TITLE
RD-5598 Add UI testing guidelines link to PR template and update main documentation file

### PR DESCRIPTION
**Pull request by qooban**
## Description

Adjusted PR template according to onboarding phase findings and updated `README.md`:
* changed main environment setup instructions (removed automated way setup and added links to docker setup page in Confluence)
* updated screenshot
* added list of platforms supported for development
* removed unused files

Ref.: RD-5598 & [Gaps in UI onboarding](https://docs.google.com/document/d/19K2PZvyfVmSlE8jd2oY6QAlamTzECpv2Q79xN4Mgtzk)

## Screenshots / Videos
N/A

## Checklist
- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
N/A

## Documentation
N/A

### CODE REVIEW

File: scripts/loadLatestPremiumDockerImage.sh [hunk 1]
```diff
@@ -1,32 +0,0 @@
-#!/usr/bin/env bash
```
Comment (kubama) [inline scripts/loadLatestPremiumDockerImage.sh:1]: Actually I'm using this script and find it handy - can we leave it?

Comment (qooban) [reply scripts/loadLatestPremiumDockerImage.sh:1]: Sure, if you use it I'll not remove it. 

I'm only wondering where to place it (as it's in `common` and `composer` repo as well). Probably here (in `cloudify-stage`) makes the most sense as we need to have Stage set up locally to run Composer, so WDYT? 🤔

Comment (kubama) [reply scripts/loadLatestPremiumDockerImage.sh:1]: Yes, it makes most sense to have it just here.

Comment (qooban) [reply scripts/loadLatestPremiumDockerImage.sh]: OK, reverted.

**Review by kubama — APPROVED** [review]

**Pull request merged**